app/views/publishers/settings/index.html.slim

```diff
@@ -1,20 +1,33 @@
+.single-panel--wrapper.single-panel--wrapper--large.single-panel--wrapper--short
+  .single-panel--padded-content--short-padding
+    h5= t ".header"
+    = form_for(current_publisher, url: publisher_path(current_publisher.id), method: :patch) do |f|
+      .row
+        .col-left-aligned--content
```

Comment:
I was thinking it make more sense to use the flex model in the Bootstrap based on here - https://getbootstrap.com/docs/4.0/utilities/flex/#wrap

It's pretty minor but it might make it easier in the future